<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCvWidget</class>
 <widget class="QWidget" name="QCvWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCvWidget</string>
  </property>
  <widget class="QLabel" name="labelViewOriginal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>571</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelViewProcessed">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>571</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelBin">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid</string>
   </property>
   <property name="text">
    <string>Binary Threshold</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>570</y>
     <width>1241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>580</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jo√£o Costa | Aveiro | 31 Jan 2021</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBinThresh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>460</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxHmin">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelHue">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid</string>
   </property>
   <property name="text">
    <string>Hue Threshold</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxHmax">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelHmin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHmax">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSmin">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSmax">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSmin">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxSmax">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSat">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid</string>
   </property>
   <property name="text">
    <string>Saturation Threshold</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelVmin">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelVmax">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxVmin">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxVmax">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>480</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelValue">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid</string>
   </property>
   <property name="text">
    <string>Value Threshold</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>530</y>
     <width>1141</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="lineEditStream">
      <property name="placeholderText">
       <string>Enter here a stream source Examples: For webcam --&gt; 0  |  For rtsp stream--&gt; rtsp://192.168.1.123/stream?user=admin&amp;psw=admin</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonPlay">
      <property name="text">
       <string>Play</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Color Scheme</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxColorScheme">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="currentText">
    <string>RGB</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
   <item>
    <property name="text">
     <string>RGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Grayscale</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HSV</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBoxBmin">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRmin">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelRmax">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGmin">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelBlue">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid</string>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelBmax">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxBmax">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>480</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRed">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid</string>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelGmax">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Max</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxGmin">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>450</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelGreen">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>430</y>
     <width>191</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid</string>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignHCenter|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxRmax">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>480</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelRmin">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxGmax">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>480</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Set here the threshold value [1,255] or 0 to disable filter</string>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelBmin">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>460</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>min</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxBinFilterOn">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>83</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <zorder>labelBin</zorder>
  <zorder>labelBlue</zorder>
  <zorder>labelGreen</zorder>
  <zorder>labelRed</zorder>
  <zorder>labelSat</zorder>
  <zorder>labelValue</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>labelHue</zorder>
  <zorder>labelViewOriginal</zorder>
  <zorder>labelViewProcessed</zorder>
  <zorder>line</zorder>
  <zorder>label_4</zorder>
  <zorder>spinBoxHmin</zorder>
  <zorder>spinBoxHmax</zorder>
  <zorder>labelHmin</zorder>
  <zorder>labelHmax</zorder>
  <zorder>labelSmin</zorder>
  <zorder>labelSmax</zorder>
  <zorder>spinBoxSmin</zorder>
  <zorder>spinBoxSmax</zorder>
  <zorder>labelVmin</zorder>
  <zorder>labelVmax</zorder>
  <zorder>spinBoxVmin</zorder>
  <zorder>spinBoxVmax</zorder>
  <zorder>label_15</zorder>
  <zorder>comboBoxColorScheme</zorder>
  <zorder>spinBoxBmin</zorder>
  <zorder>spinBoxRmin</zorder>
  <zorder>labelRmax</zorder>
  <zorder>labelGmin</zorder>
  <zorder>labelBmax</zorder>
  <zorder>spinBoxBmax</zorder>
  <zorder>labelGmax</zorder>
  <zorder>spinBoxGmin</zorder>
  <zorder>spinBoxRmax</zorder>
  <zorder>labelRmin</zorder>
  <zorder>spinBoxGmax</zorder>
  <zorder>labelBmin</zorder>
  <zorder>checkBoxBinFilterOn</zorder>
  <zorder>spinBoxBinThresh</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
